geotrek/common/models.py

```diff
@@ -264,3 +270,136 @@ def __str__(self):
 
     class Meta:
         abstract = True
+
+
+class HDViewPoint(TimeStampedModelMixin):
+    picture = models.FileField(verbose_name=_("Picture"), upload_to="hdviewpoints/")
+    geom = gis_models.PointField(verbose_name=_("Location"),
+                                 srid=settings.SRID)
+    object_id = models.PositiveIntegerField()
+    content_type = models.ForeignKey(ContentType, on_delete=models.CASCADE)
+    content_object = GenericForeignKey('content_type', 'object_id')
+    annotations = models.JSONField(null=True, verbose_name=_("Annotations"), blank=True)
```

Comment:
```suggestion
    annotations = models.JSONField(null=False, verbose_name=_("Annotations"), blank=True, default=dict)
```
I suggest to use default=dict here